packages/web/src/common/store/social/users/sagas.ts

```diff
@@ -73,6 +77,15 @@ export function* followUser(
   yield* put(event)
 
   yield* call(confirmFollowUser, action.userId, accountId)
+  const shouldAutoSubscribe =
+    ((yield* call(getFeatureEnabled, FeatureFlags.AUTO_SUBSCRIBE_ON_FOLLOW)) as
```

Comment:
Anyway this can be a little less nested?

Reply:
kinda not sure what the best pattern is b/c the getFeatureEnabled stuff doesn't seem typed

i copied dis one https://github.com/AudiusProject/audius-client/blob/edc84337702102d4e460eaf508a24a1974f76949/packages/web/src/common/store/notifications/sagas.ts#L140

Reply:
chatted offline -- will look into this but at a later date! agreed bad!!